tensorflow/compiler/tests/adam_test.py

```diff
@@ -52,6 +52,8 @@ class AdamOptimizerTest(XLATestCase):
 
   def testBasic(self):
     for dtype in self.float_types:
+      if dtype == np.float16:
```

Comment:
It would be better to fix the test (either by relaxing the test tolerance or improving the test in some other way).

Would you mind either:
a) changing the float16 tolerance (assertAllCloseAccordingToType allows a per-type tolerance override) to make the tests pass
or
b) adding a comment along the lines of
# TODO: test fails for float16 due to excessive precision requirements.
so it is clear why the test is disabled?

Reply:
Sorry for delay, done.